src/App.js

```diff
@@ -0,0 +1,11 @@
+import './styles/App.css';
+import React from 'react';
+import Router from './routes/routes';
+
+const App = () => (
```

Comment:
@Meltrust, Thanks for your hard work and for giving us a detailed review. Kindly, can I ask for more details, like do you mean that when you click on the `Logout` button the onClick method is not working and to redirect to the root page you have to delete the local Storage? It would help us understand more cause, currently, when I click on the logout button, the thunk method is being called and the local storage variables are also being removed.
Here is the preview: 
https://clipchamp.com/watch/r8tHlRT19uQ

Reply:
Thanks for taking the time and re-checking, we appreciate it!☺️🎊